conf/application.conf

```diff
@@ -175,6 +175,8 @@ allowlist = ""
 allowlist-referrers = ""
 # comma separated list of uri to block
 routes-to-block = ""
+# TODO - change to 31/12
+displayEuLocationUntil = "2020-10-31T23:00+00:00"
```

Comment:
Should this be done to the correct value now?

With correct value we won't need to override this in all environments (but still production is worth to override just in case).

I'm just wondering what about the development environments, but I think on development environment for test purposes we need to imitate the situation after 31/12/2020, because some of the acceptance tests require fixes, so we need to merge everything

Reply:
My logic was to enable the change everywhere except Prod and ET (with this already-passed date).     So that all testing done from now on will be on the "to-be" FE.    You are right about getting rid of the TODO comment though.    There is no point changing the date later on.